### DISCUSSION TITLE
On the management page,click Admin,and then click Users to get an error

### DISCUSSION THREAD

**Discussion by Joe0612** [rabbitmq/rabbitmq-server#9661] [Other]
### Describe the bug

The rabbitmq version is 3.9.28, erlang version is 24.3.4.13, go to the management page, click Admin, then click Users error message.
The error message is as follows
`SyntaxError: Unexpected token 'catch' SyntaxError: Unexpected token 'catch' at EJS.Compiler.compile (http://localhost:15672/js/ejs-1.0.min.js:1:6659) at new EJS (http://localhost:15672/js/ejs-1.0.min.js:1:1625) at format (http://localhost:15672/js/main.js:1164:20) at http://localhost:15672/js/main.js:486:24 at with_reqs (http://localhost:15672/js/main.js:1147:9) at http://localhost:15672/js/main.js:1143:17 at req.onreadystatechange (http://localhost:15672/js/main.js:1238:17)`

I checked the official documents,the rabbitmq version is suitable to the erlang version.I don't konw how to slove this problem

### Reproduction steps

1.Go to the management page.http://localhost:15672
2.Click Admin.
3.Click Users.
...

### Expected behavior

I want to know how to slove this problem.

**Comment by michaelklishin** [comment]
RabbitMQ 3.9 has reached [end of life](https://rabbitmq.com/versions.html).

I cannot reproduce in cutting edge Chrome, Firefox, Safari. Chances are you have upgraded to 3.9 from an earlier version and [need to clear your browser's cache](https://rabbitmq.com/upgrade.html#rabbitmq-plugins-compatibility).

See this [very similar rabbitmq-users thread](https://groups.google.com/g/rabbitmq-users/c/uDoO6tMP6Lg), for example.
